flight/targets/pipxtreme/board-info/pios_board.h

```diff
@@ -165,22 +165,18 @@ extern uint32_t pios_i2c_flexi_adapter_id;
 //
 // See also pios_board.c
 //-------------------------
-extern uintptr_t pios_com_telem_usb_id;
-extern uintptr_t pios_com_telem_vcp_id;
-extern uintptr_t pios_com_telem_uart_telem_id;
-extern uintptr_t pios_com_telem_uart_main_id;
-extern uintptr_t pios_com_telemetry_id;
-extern uintptr_t pios_com_rfm22b_id;
-extern uintptr_t pios_com_radio_id;
+extern uintptr_t pios_com_telem_usb_id; 
+extern uintptr_t pios_com_vcp_id; 
+extern uintptr_t pios_com_radio_id; 
+extern uintptr_t pios_com_telem_rf_id; 		// POORLY NAMED
```

Comment:
note this is WIP and on my task list above is variable name cleanup and conditionalization of these...